tools/spdx-check/spdx-check.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+# Copyright (c) 2024 Zededa, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+BASE_COMMIT=$1
+SCRIPT_DIR=$(dirname "$0")
+IGNORE_FILE="$SCRIPT_DIR/.spdxignore"
+
+# List of files to check, excluding vendor directories
+files=$(git diff --name-only --diff-filter=A "${BASE_COMMIT}"..HEAD)
+
+# SPDX License Identifier to check for
+license_identifiers=(
+  "Apache-2.0"
+  "MIT"
+  "BSD-2-Clause"
+  "BSD-3-Clause"
+  "ISC" # Internet Systems Consortium license
+  "Python-2.0"
+  "PostgreSQL"
+  "Zlib"
+  "BSL-1.0" # Boost Software License 1.0
+  "CC0-1.0" # Creative Commons Zero v1.0 Universal
+  "WTFPL" # Do What The F*ck You Want To Public License
```

Comment:
This one is not OSI approved and it's basically Public Domain, we don't want that, one of the premises of FOSS licenses is exactly to keep the Copyright to the authors, so they, in their Copyright holder rights, can impose the freedom to Use / Change / Redistribute as the licensees wish....

Reply:
Saaaad T_T
Ok, I'll remove

Reply:
Done